autonomi/src/client/data_types/chunk.rs

```diff
@@ -39,7 +39,7 @@ pub(crate) static CHUNK_UPLOAD_BATCH_SIZE: LazyLock<usize> = LazyLock::new(|| {
     let batch_size = std::env::var("CHUNK_UPLOAD_BATCH_SIZE")
         .ok()
         .and_then(|s| s.parse().ok())
-        .unwrap_or(1);
+        .unwrap_or(64);
```

Comment:
It is the most robust one preferred.
increasing this value has proved causeing problem to user with normal connection